npm/private/utils.bzl

```diff
@@ -72,13 +71,12 @@ def _package_store_name(pnpm_name, pnpm_version):
         return "%s@%s" % (escaped_name, escaped_version)
 
 def _make_symlink(ctx, symlink_path, target_path):
-    if not bazel_lib_utils.is_bazel_6_or_greater():
-        # ctx.actions.declare_symlink was added in Bazel 6
-        fail("A minimum version of Bazel 6 required to use rules_js")
     symlink = ctx.actions.declare_symlink(symlink_path)
+    new_target_path = relative_file(target_path, symlink.path)
+
     ctx.actions.symlink(
         output = symlink,
-        target_path = relative_file(target_path, symlink.path),
+        target_path = new_target_path,
```

Comment:
There's no actual change here?

Reply:
Can we revert this and rebase the PR to minimize the diff here?

Reply:
yes, I'll do that once we decide if we're landing the paths -> str.format PR